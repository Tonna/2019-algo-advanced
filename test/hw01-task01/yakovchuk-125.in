100
21 32 69 29 85 41 54 53 5 53 78 19 8 28 64 22 51 24 26 4 91 1 11 13 5 52 76 87 97 60 54 10 29 61 30 67 45 11 46 50 16 49 16 28 59 75 13 15 49 14 47 97 66 80 88 25 92 14 47 77 34 82 69 43 23 94 9 57 13 91 31 40 50 8 34 60 3 20 96 9 86 14 12 94 47 40 63 65 54 96 56 62 13 96 5 55 79 84 17
1
1 1
