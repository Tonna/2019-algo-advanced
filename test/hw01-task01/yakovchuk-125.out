25012
17243
8687
42657
109862
59133
73871
24766
